pkg/shell/topnav.jsx

```diff
@@ -123,11 +123,15 @@ export class TopNav extends React.Component {
 
         if (!this.superuser_connection || (this.superuser_connection.options.host !=
                                            this.props.machine.connection_string)) {
-            if (this.superuser_connection)
+            if (this.superuser_connection) {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18991-20230628-121551-cf6ae764-fedora-38-devel-storage//Coverage/lcov/github-pr.diff.gcov.html)